python/src/aiconfig/editor/client/src/components/prompt/AddPromptButton.tsx

```diff
@@ -79,8 +79,9 @@ export default memo(function AddPromptButton({ addPrompt, getModels }: Props) {
           collapseLimit={5}
           onSelectModel={onAddPrompt}
         />
+        {/* TODO: Add back once we have custom model parsers fully supported
         <Menu.Divider />
-        <Menu.Item icon={<IconPlus size="16" />}>Add New Model</Menu.Item>
+        <Menu.Item icon={<IconPlus size="16" />}>Add New Model</Menu.Item> */}
```

Comment:
This makes sense. Also my feedback for this (once we add it back) is that it's confusing. It's not immediately clear what "Add new model" should do here. Wondering if this should belong in a toolbar at the top alongside save. cc @zakariassoul

Reply:
@saqadri the button wasn't actually implemented fully -- designs show that it will show an info modal on how to add a custom model parser which will then show up in the actual model list once added following the instructions. I just didn't implement the modal yet because it would not have correct info (since we don't support custom model parsers in the editor just yet)